img/foxsi_image_map.pro

```diff
@@ -45,7 +45,7 @@ default, thr_n, 4.		; n-side keV threshold
 default, year, 2014
 default, trange, [0,500]
 default, fov, 20.
-
+default, flatfield, 0
 	xc = center[0]
 	yc = center[1]
 	
```

Comment:
Should the default for CdTe be set to 0?

Reply:
No because I have the mention "if keyword_set( CDTE ) ..." in line 66, so if I set a default value I would have to change this. I can be done but it should work the same as it is now.
The other place where the keyword CdTe is mentioned is when calling foxsi_image_det, in which there is a default value of 0 for the cdte keyword.